Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -98,9 +98,7 @@ extension SPAppDelegate {
         guard let simperiumKey = url.interlinkSimperiumKey, let note = simperium.loadNote(simperiumKey: simperiumKey) else {
             return false
         }
-
-        navigationController.presentedViewController?.dismiss(animated: true, completion: nil)
-        noteListViewController.open(note, ignoringSearchQuery: true, animated: true)
```

Comment:
There must be a reason why we pass `ignoringSearchQuery: true` here, though I don't remember why :-)
To keep this logic we can do the following:
```
popToNoteList()
noteListViewController.open(note, ignoringSearchQuery: true, animated: true)
```

Reply:
Fixed